Modules/Relationships/src/Relationships.Domain/Aggregates/Relationships/Relationship.cs

```diff
@@ -155,14 +155,16 @@ public void RevokeReactivation(IdentityAddress activeIdentity, DeviceId activeDe
         AuditLog.Add(auditLogEntry);
     }
 
-    private void EnsureOpenReactivationRequestExists(IdentityAddress target)
+    private void EnsureOpenReactivationRequestExists(IdentityAddress activeIdentity)
```

Comment:
The method name is misleading. One might use it e.g. for the rejection of a reactivation request. But there the requirement is that the reactivation must NOT to be created by the active identity.

A better name would maybe be `EnsureRevocableReactivationRequestExistsFor`. But I'm open to other suggestions.

Reply:
No better idea, I used yours.

Changes made.